### PR TITLE
Added Percentage column with list

### LINKED ISSUE #126: Show percentage
While using the app, I found that sometimes I need to finish pomodoro timer early. In that case, I want to record how much did I spend - 70%, 50% for example. Let's make this option

This is adding new column named percentage

### PR DESCRIPTION
**Pull request by TheRustyPickle**
**Updates:**

- Fixed `create` and `queue` command not handling `-w` and`-b` and instead was using default values for all cases due to `contains_id` method returning `true`.
- Added `percentage` column when `-p` flag is passed with `ls`

Note: When using UDS, the app does not show the percentage column yet. Resolves #126.

### CODE REVIEW

File: src/command/application.rs [hunk 1]
```diff
@@ -105,7 +105,8 @@ fn get_common_subcommands() -> Vec<Command> {
             ),
         Command::new(ActionType::List)
             .alias(LS)
-            .about("list notifications"),
+            .about("list notifications")
+            .arg(Arg::new("percentage").short('p').num_args(0)),
```
Comment (24seconds) [inline src/command/application.rs:109]: [Suggestion]
How about writing description for this argument?

**Review by 24seconds — COMMENTED** [review]
Looks good overall. Thank you for the contribution. I left some comments. Could you take a look?

File: src/command/handler/user_input.rs [hunk 2]
```diff
@@ -254,22 +255,31 @@ async fn handle_test(
...
 ) -> HandleUserInputResult {
     debug!("Message::List called!");
     let notifications = db::list_notification(glue.clone()).await;
     debug!("Message::List done");
 
-    let table = notifications
-        .table()
+    let mut main_table = notifications.table();
+
+    let table_builder = main_table
```
[the remark this answers is not in the document]
Comment (TheRustyPickle) [reply src/command/handler/user_input.rs:268]: Would you happen to have any suggestions for it? Perhaps `table_formatter`, `styled_table`, or `modified_table`?

File: src/command/application.rs (hunk 1 above)
Comment (TheRustyPickle) [reply src/command/application.rs:109]: How about this as a description: show work time completion percentage

File: src/command/handler/user_input.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (24seconds) [reply src/command/handler/user_input.rs:268]: styled_table or modified_table sounds good for me

File: src/command/application.rs (hunk 1 above)
Comment (24seconds) [reply src/command/application.rs:109]: good!

**Pull request closed without merging**